data/json/mapgen/nested/retail_nested.json

```diff
@@ -0,0 +1,1369 @@
+        "B": "f_counter",
+        "C": "f_table",
+        "G": "f_displaycase",
+        "b": "f_bigmirror",
+        "Q": "f_rack_wood",
+        "d": "f_mannequin",
+        "L": "f_wardrobe"
+      },
+      "items": {
+        "d": [
+          { "item": "fancyfurs", "chance": 100 },
+          { "item": "pants", "chance": 50 },
+          { "item": "shirts", "chance": 50 },
+          { "item": "dress_shoes", "chance": 100 }
+        ],
+        "C": [ { "item": "fancyfurs", "chance": 30, "repeat": [ 1, 2 ] } ],
+        "L": [ { "item": "fancyfurs", "chance": 30, "repeat": [ 1, 2 ] } ],
+        "Q": [ { "item": "fancyfurs", "chance": 30, "repeat": [ 1, 2 ] } ],
+        "G": [
+          { "item": "neckties", "chance": 30, "repeat": [ 0, 4 ] },
+          { "item": "jewelry_front", "chance": 30, "repeat": [ 0, 1 ] }
+        ]
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "designed for 4x4 fur clothing shop crates.",
+    "nested_mapgen_id": "city_block_4x4_backroom_clothing_fur",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "z  z",
+        " z  ",
+        "z z ",
+        "z  z"
+      ],
+      "terrain": { "z": "t_floor" },
+      "furniture": { "z": [ [ "f_cardboard_box", 5 ], "f_crate_c" ] },
+      "items": {
+        "z": [
+          { "item": "fancyfurs", "chance": 30, "repeat": [ 1, 6 ] },
+          { "item": "neckties", "chance": 30, "repeat": [ 1, 6 ] },
+          { "item": "accesories_personal_unisex", "chance": 30, "repeat": [ 1, 6 ] }
+        ]
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "designed for 4x4 fur clothing backroom.",
+    "nested_mapgen_id": "city_block_4x4_backroom_clothing_fur",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "CCCC",
+        "    ",
+        "    ",
+        "CC C"
+      ],
+      "terrain": { " ": "t_floor" },
+      "furniture": { "C": "f_rack" },
+      "items": { "C": [ { "item": "fancyfurs", "chance": 30, "repeat": [ 0, 6 ] } ] }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "designed for 4x4 fur clothing backroom.",
+    "nested_mapgen_id": "city_block_4x4_backroom_clothing_fur",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "CBBB",
+        "  Ad",
+        "A   ",
+        "BB C"
+      ],
+      "terrain": { " ": "t_floor", "?": "t_console_broken" },
+      "furniture": { "A": "f_stool", "B": "f_workbench", "C": "f_rack_wood", "d": "f_mannequin", "e": "f_mannequin", "b": "f_bench" },
+      "items": {
+        "C": [
+          { "item": "SUS_tailoring_materials", "chance": 10, "repeat": [ 1, 2 ] },
+          { "item": "SUS_fur_tailoring_materials", "chance": 30, "repeat": [ 2, 6 ] }
+        ],
+        "d": [ { "item": "SUS_mannequin_fur", "chance": 100 } ]
+      },
+      "B": [
```

Comment:
Looks like a typo here, I don't think you meant to end the items list before the "B" entry.

Reply:
thanks!